src/components/application_manager/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -194,6 +194,19 @@ bool ResetGlobalPropertiesRequest::ResetHelpPromt(
   }
   smart_objects::SmartObject so_help_prompt =
       smart_objects::SmartObject(smart_objects::SmartType_Array);
+
+  const std::vector<std::string>& help_prompt_default =
+      application_manager_.get_settings().help_prompt();
+
+  for (uint32_t i = 0; i < help_prompt_default.size(); ++i) {
```

Comment:
@AKalinich-Luxoft Don't compare different types even both of them unsigned. Type sizes can be different.